packages/access-blob-storage/lib/minio/MinioPreviewStorage.mjs

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-unused-vars */
+import {PreviewStorage} from '../core'
+
+export class MinioPreviewStorage extends PreviewStorage {
+  #storage = null
+  #bucket = 'preview'
+
+  /**
+   * @param storage - Minio storage
+   */
+  constructor(storage) {
+    super()
+    this.#storage = storage
+  }
+
+  async save(params) {
+    return await this.#storage.putObject(this.#bucket, params.name, params.data)
```

Comment:
We should put some effort in 'wrapping' underling api of minio

for example 'just' returning original response from minio is not enought we should return our format of response, it
even became more interesting that we also need to take care for `errors`, so each call for underlined api should try to handle not only happy path but failure too